package.json

```diff
@@ -46,23 +51,45 @@
 			"@commitlint/config-conventional"
 		]
 	},
+	"files": [
+		"dist/"
+	],
+	"exports": {
+		".": "./dist/cosmoz-treenode.js",
+		"./cosmoz-treenode": "./dist/cosmoz-treenode.js",
+		"./cosmoz-treenode.js": "./dist/cosmoz-treenode.js"
+	},
 	"husky": {
 		"hooks": {
 			"commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
 		}
 	},
 	"dependencies": {
-		"@neovici/cosmoz-tree": "^3.0.0",
-		"@pionjs/pion": "^2.0.0"
+		"@neovici/cosmoz-tree": "^3.6.0",
+		"@pionjs/pion": "^2.5.2"
 	},
 	"devDependencies": {
-		"@commitlint/cli": "^18.0.0",
-		"@commitlint/config-conventional": "^18.0.0",
-		"@neovici/cfg": "^1.15.2",
+		"@commitlint/cli": "^19.2.1",
+		"@commitlint/config-conventional": "^19.1.0",
+		"@neovici/cfg": "^1.50.0",
 		"@open-wc/testing": "^4.0.0",
 		"@semantic-release/changelog": "^6.0.0",
 		"@semantic-release/git": "^10.0.0",
-		"husky": "^8.0.0",
-		"semantic-release": "^22.0.0"
+		"@storybook/addon-essentials": "^7.0.0",
+		"@storybook/addon-links": "^7.0.0",
+		"@storybook/builder-vite": "7.6.17",
+		"@storybook/storybook-deployer": "^2.8.5",
+		"@storybook/web-components": "7.6.17",
+		"@types/mocha": "^10.0.6",
+		"@web/storybook-builder": "^0.1.9",
```

Comment:
Isn't `@web/storybook-builder` and `@web/storybook-framework-web-components` enough ? Do we need to add all the `@storybook/*` plugin to deps?